src/action/Action.hpp

```diff
@@ -17,7 +17,7 @@ class Action {
 public:
   /// Defines the time and place of application of the action.
   enum Timing {
-    ON_TIME_WINDOW_COMPLETE_POST, // On advancing to next dt, after adv. cpl scheme
+    ON_TIME_WINDOW_COMPLETE_POST, // On advancing to next time step, after adv. cpl scheme
```

Comment:
Are you sure here? Meaning that it is not "time window"? Does the action get called after each timestep or only after the complete time window?

Reply:
```suggestion
    ON_TIME_WINDOW_COMPLETE_POST, // On advancing to next time window, after adv. cpl scheme
```
From the name `ON_TIME_WINDOW_COMPLETE_POST` the comment should also refer to the time window. Yes.

Reply:
@fsimonis Does this make sense?

Reply:
It is called when `_couplingScheme->isTimeWindowComplete())`, so that makes sense.

Reply:
To clarify, `time window` makes sense :wink: